### PR TITLE
Add names to schema from git_table

### PR DESCRIPTION
**Pull request by michaelquinn32**
This makes it easier to explore the schema in the Rstudio Viewer.

### CODE REVIEW

File: R/tables.r [hunk 1]
```diff
@@ -70,7 +70,21 @@ get_table <- function(project, dataset, table) {
   assert_that(is.string(project), is.string(dataset), is.string(table))
 
   url <- sprintf("projects/%s/datasets/%s/tables/%s", project, dataset, table)
-  bq_get(url)
```
Comment (craigcitro) [inline R/tables.r]: I suspect something was dropped here -- did you want this to be

    metadata <- bq_get(url)

instead?

Comment (michaelquinn32) [reply R/tables.r]: Oops. It's put back.

**Comment by byapparov** [comment]
Do you mind adding the tests?

Pull request is about get_table, not git_table 

It does not make sense to change the output of the ‘get_tables’ as one could use it to create similar tables (without tests its difficult to assert what are you trying to achieve).

If you want pretty output in console, there should be another function, e.g. summary() or print.schema() for that.

**Comment by hadley** [comment]
I agree with @byapparov. Can you please start by filing an issue that describes the motivating problem?

**Pull request closed without merging**